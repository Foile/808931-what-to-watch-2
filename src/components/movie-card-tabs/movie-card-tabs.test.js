import React from "react";
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import MovieCardTabs from "./movie-card-tabs";

it(`MovieCardTabs correctly renders after relaunch`, () => {
  const movie = {id: 1, name: `Macbeth`, previewImage: ``, previewVideoLink: ``, starring: []};
  const tree = renderer
  .create(<BrowserRouter>
    <MovieCardTabs
      movie = {movie}
      match = {{params: {id: 1}}} />
  </BrowserRouter>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
