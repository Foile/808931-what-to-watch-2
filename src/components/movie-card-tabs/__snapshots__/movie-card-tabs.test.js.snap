// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardTabs correctly renders after relaunch 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="/films/1/overview"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          href="/films/1/details"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          href="/films/1/reviews"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    />
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        
         and other
      </strong>
    </p>
  </div>
</div>
`;
